radicle-node/src/client.rs

```diff
@@ -73,24 +89,27 @@ impl<R: Reactor> Client<R> {
         })
     }
 
-    pub fn run(mut self, config: Config) -> Result<(), nakamoto_net::error::Error> {
+    pub fn run(mut self, config: Config) -> Result<(), Error> {
         let network = config.service.network;
         let rng = fastrand::Rng::new();
         let time = LocalTime::now();
         let storage = self.profile.storage;
         let signer = self.profile.signer;
         let addresses = HashMap::with_hasher(rng.clone().into());
+        let routing = routing::Table::open("routing.db")?;
```

Comment:
Would making it a configuration based path make it better?  Like path::join(config.data, "routing.db")

Reply:
Ah yes, forgot to do this, good catch!